config/config.php

```diff
@@ -11,9 +11,9 @@
     |
     | Select how a user account should be activated.
     |
-    | ACTIVATE_ADMIN    Only an Administrator can activate a user.
-    | ACTIVATE_AUTO     Activated automatically upon registration.
-    | ACTIVATE_USER     The user activates their own account using mail.
+    | ACTIVATE_ADMIN    Administrators must activate users manually.
+    | ACTIVATE_AUTO     Users are activated automatically upon registration.
+    | ACTIVATE_USER     The user activates their own account using a link sent to them via email.
```

Comment:
Heh, thanks for this @LukeTowers. I just copied values out of the english language file, I probably could have made them read better given the different context 😅